go.mod

```diff
@@ -5,8 +5,9 @@ go 1.16
 require (
 	github.com/Masterminds/semver/v3 v3.1.1
 	github.com/ghodss/yaml v1.0.0
-	github.com/giantswarm/apiextensions/v3 v3.22.0
+	github.com/giantswarm/apiextensions/v3 v3.26.0
 	github.com/giantswarm/app/v4 v4.13.0
+	github.com/giantswarm/app/v5 v5.0.0-20210603090121-ada93de65e7f
```

Comment:
TODO; update this to 5.0.0 before merge

Reply:
I will not release a new app-operator version until we get rid github.com/giantswarm/app/v4